fathom_fox/addon/pages/evaluate.html

```diff
@@ -133,14 +133,14 @@ <h1>Evaluator</h1>
       No rulesets found. Please download a source checkout of FathomFox, and define one in rulesets.js.
     </p>
     <p>
-      Evaluate your ruleset based on the samples you’ve loaded as tabs in this window.
+      Evaluate your ruleset, for the selected trainee, against the samples you’ve loaded as tabs in this window.
     </p>
     <div class="uiGrid">
       <div class="label">
-        <label for="ruleset">Ruleset:</label>
+        <label for="trainee">Trainee ID:</label>
       </div>
       <div>
-        <select id="ruleset"></select>
+        <select id="trainee"></select>
```

Comment:
This probably breaks initRulesetMenu(), which finds the menu by ID.

Reply:
Indeed it does.